catalogs/data/README.md

```diff
@@ -0,0 +1,11 @@
+# Supported Catalogs
+The following catalogs are sourced from https://github.com/CMSgov/ars-machine-readable .
+These catalogs are ingested into the blueprint database during deployment, and should not be modified. 
+Projects should be associated with a single catalog dataset given a catalog version and impact level 
+("high", "moderate", "low").
+
+## Version 3.1 (current)
+- CMS_ARS_3_1_[HIGH, MODERATE, LOW]_baseline_catalog.json
+
+## Version 5.0
+- cms_ars_50_[high, moderate, low].jon
```

Comment:
Forgot to remove this. I'll drop this section. 

Let me know if more detail should be added here. @TMCamp might know what would be most useful here.